### PR TITLE
Create get_result_from_api.gs

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: get_result_from_api.gs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    var endTime = (new Date()).getTime();
+    if (endTime - startTime > 25000) {
+      var card = refreshScreenUI(e, token, limitResponse, reportTitle, urlEnd);
+      var navigation = CardService.newNavigation().updateCard(card);
+      var actionResponse =
+        CardService.newActionResponseBuilder().setNavigation(navigation);
+      return actionResponse.build();
+    }
+    Utilities.sleep(5000);
```
Comment (shikhagupta1212) [inline get_result_from_api.gs:38]: why sleep?

File: get_result_from_api.gs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  };
+  while (result == null) {
+    var response = UrlFetchApp.fetch(url, options);
+    var json = response.getContentText();
+    result = JSON.parse(json);
+    if (result) {
+      break;
+    }
+    var endTime = (new Date()).getTime();
+    if (endTime - startTime > 25000) {
```
Comment (shikhagupta1212) [inline get_result_from_api.gs:31]: Why this check

File: get_result_from_api.gs (hunk 1 above)
Comment (siddhimahawar) [reply get_result_from_api.gs:38]: To reduce the number of API calls
This call is to check whether the result we asked for is ready or not

File: get_result_from_api.gs (hunk 2 above)
Comment (siddhimahawar) [reply get_result_from_api.gs:31]: The callback functions have less execution time than the whole apps script
They show maximum execution error in about 25-30 seconds so before that only we can show the refresh screen

**Review by piy314 — APPROVED** [review]

**Pull request merged**